modules/ROOT/pages/type-definitions/directives/index.adoc

```diff
@@ -1,11 +1,182 @@
 [[directives]]
 :description: This page lists all directives available in the Neo4j GraphQL Library.
+
 = Directives
 :page-aliases: directives.adoc
 
 The Neo4j GraphQL Library provides the following directives to be used whilst defining types:
 
-[cols="1,3"]
+== Database mapping
+
+.Database mapping directives
+[cols="2,5"]
+|===
+| Directive | Description
+
+| xref::/type-definitions/directives/database-mapping.adoc#_relationship[`@relationship`]
+| Configure  xref::/type-definitions/types/relationships.adoc[relationships] between object types.
```

Comment:
Most verbs are in the third person, but all sentences starting with "configure" seem to be imperative. Is that intended?